HTML and CSS/Product listing page with filters/product-listing.css

```diff
@@ -0,0 +1,108 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@200;300;400;600&family=Nunito:wght@300;400;500;600;700&display=swap');
+
+
+* {
+    box-sizing: border-box;
+    padding: 0;
+    margin: 0;
+    font-family: 'Montserrat', sans-serif;
+    font-family: 'Nunito', sans-serif;
+}
+
+body{
+    background-color: #f1f3f6;
+}
+
+.sort-byname {
+    border: none;
+    font-size: 0.9rem;
+    font-weight: 600;
+    padding: 0rem 0.5rem;
+}
+
+.sort-sub {
+    font-size: 0.8rem;
+    font-weight: 500;
+    padding: 0rem 0.5rem;
+}
+
+.filter-text {
+    font-size: 0.88rem;
+    font-weight: 400;
+}
+
+.sortby-container {
+    border: 1px solid gainsboro;
+    box-shadow: 0.1px 0.1px 0.2px 0.6px gainsboro;
+    width: max-content;
+    padding: 0.5rem 1rem;
+    background-color: white;
+}
+
+.all-contain {
```

Comment:
A hover element here might help as well , and to the specific checkboxes as well maybe to make it look more interractive maybe ? 
Onkar Deshpande